src/openforms/formio/tests/validation/test_map.py

```diff
@@ -0,0 +1,30 @@
+from django.test import SimpleTestCase
+
+from ...typing import Component
+from .helpers import extract_error, validate_formio_data
+
+
+class MapValidationTests(SimpleTestCase):
+
+    def test_map_field_required_validation(self):
+        component: Component = {
+            "type": "map",
+            "key": "foo",
+            "label": "Test",
+            "validate": {"required": True},
+        }
+
+        invalid_values = [
+            ({}, "required"),
+            ({"foo": ""}, "not_a_list"),
+            ({"foo": None}, "null"),
+        ]
```

Comment:
I'd additionally check that `[<number>]` and `[<number1>, <number2>, <number3>]` are not accepted - you can specify min/max number of items on the serializer list field if I remember correctly.